components/web/apps/org.wso2.carbon.apimgt.admin.app/src/main/lang/en_US.properties

```diff
@@ -63,4 +63,23 @@ Execution.Policy = Execution Policy
 Throttle.based.on.request.count = Throttle based on request count
 Throttle.Based.on.bandwidth = Throttle Based on bandwidth.
 Bandwidth = Bandwidth
-Data.Bandwidth = Data Bandwidth
\ No newline at end of file
+Data.Bandwidth = Data Bandwidth
+add.subscription.tier = Add Subscription Tier
```

Comment:
Key should be same as the Value. ie:
Add.Subscription.Tier = Add Subscription Tier